zebra-state/src/service/read/difficulty.rs

```diff
@@ -82,6 +82,7 @@ pub fn get_block_template_chain_info(
 ///
 /// Returns the solution rate per second for the current best chain, or `None` if
 /// the `start_hash` and at least 1 block below it are not found in the chain.
+#[allow(unused)]
```

Comment:
```suggestion
```

I am not sure if this is needed for some build type, i commented out and built with `cargo build` and there were no complains.